README.md

```diff
@@ -204,6 +208,56 @@ The following sections provide a full list of configuration in- and output varia
 | vault\_user\_id | The Vault IAM user id |
 | vault\_user\_secret | The Vault IAM user secret |
 
+### Cluster Autoscaling
+
+This does not automatically install cluster-autoscaler, it installs all of the prerequisite policies and roles required to install autoscaler. The actual autoscaler installation varies depending on what version of kubernetes you are using.
```

Comment:
We don't enforce it yet, but would be nice to have one sentence per line in the markdown.